src/components/chart/_macro.spec.js

```diff
@@ -879,6 +881,186 @@ describe('Macro: Chart', () => {
         });
     });
 
+    describe('FOR: Boxplot Chart', () => {
+        describe('GIVEN: Params: required', () => {
+            describe('WHEN: required params are provided', () => {
+                const $ = cheerio.load(renderComponent('chart', EXAMPLE_BOXPLOT_CHART_PARAMS));
+                const configScript = $(`script[data-highcharts-config--uuid]`).html();
+
+                test('THEN: it passes jest-axe checks', async () => {
+                    const results = await axe($.html());
+                    expect(results).toHaveNoViolations();
+                });
+
+                test('THEN: it includes at least one boxplot series', () => {
+                    expect(configScript).toContain('"type":"boxplot"');
+                });
+
+                test('THEN: it renders the chart container with correct data attributes', () => {
+                    const chartContainer = $('[data-highcharts-base-chart]');
+                    expect(chartContainer.attr('data-highcharts-type')).toBe('boxplot');
+                    expect(chartContainer.attr('data-highcharts-theme')).toBe('primary');
+                    expect(chartContainer.attr('data-highcharts-title')).toBe(
+                        'Figure 6: Net debt as a percentage of GDP remains at levels last seen in the early 1960s',
+                    );
+                    expect(chartContainer.attr('data-highcharts-id')).toBe('uuid');
+                });
+
+                test('THEN: it includes the Highcharts JSON config', () => {
+                    expect(configScript).toContain('"text":"Years"');
+                    expect(configScript).toContain('"text":"Percentage of GDP"');
+                });
+            });
+        });
+
+        describe('GIVEN: Params: Legend', () => {
+            describe('WHEN: legend is disabled', () => {
+                const $ = cheerio.load(renderComponent('chart', { ...EXAMPLE_BOXPLOT_CHART_PARAMS, legend: false }));
+
+                test('THEN: it disables the legend in config', () => {
+                    const configScript = $(`script[data-highcharts-config--uuid]`).html();
+                    expect(configScript).toContain('"enabled":false');
+                });
+            });
+        });
+
+        describe('GIVEN: Params: Caption', () => {
```

Comment:
I have been wondering if we need to repeat the tests for caption, description etc for each chart type, given that these are the same for every chart. What do you think?

Reply:
It looks like you've removed this now so I'll resolve.